databricks/koalas/tests/test_dataframe.py

```diff
@@ -458,3 +458,16 @@ def test_merge(self):
         res = left_kdf.merge(koalas.DataFrame({'A': [3, 4]}), left_index=True, right_index=True,
                              suffixes=('_left', '_right'))
         self.assert_eq(res, pd.DataFrame({'A_left': [1, 2], 'A_right': [3, 4]}))
+
+    def test_clip(self):
+        pdf = pd.DataFrame({'A': [0, 2, 4]})
```

Comment:
i'd add a test case for non-numeric type, e.g. string.

Reply:
Done with 267e509.